product_docs/docs/pgd/4/rel_notes/pgd_4.3.3_rel_notes.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.3"
+navTitle: "Version 4.3.3"
+redirects:
+  - /pgd/latest/bdr/release_notes/bdr4.3.3_rel_notes/
+---
+
+EDB Postgres Distributed version 4.3.3 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+!!! Note
+This version is required for EDB Postgres Advanced Server versions 12.14.18, 13.10.14, 14.7.0 and later.
+!!!
+
+| Component | Version |  Type        |                                                   Description                                                           |
+| --------- | ------- | ------------ | ------------------------------------------------------------------------------------------------------------------------|
+| BDR       | 5.3.0   | Bug&nbsp;fix | Ensure that the WalSender process does not request locks on the partitions, thus avoiding a deadlock with user process waiting on sync commit. (RT97952) |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Only consider CAMO transactions to be asynchronous when the CAMO setup was degraded to local mode. This solves the split brain problem when deciding fate of transactions that happened during failover. (RT78928) |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Handle partitions with different attribute numbers when batch inserting. (RT99115) |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Fix unsafe CAMO decisions in remote_write mode. |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Ensure that the taskmgr process does not ignore SIGINT. |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Clean up stale records in bdr.taskmgr_local_work_queue during physical join. |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Fix a bug in copying bdr.autopartition_rules during logical join. |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Override bdr.ddl_replication=off in taskmgr worker. |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Avoid a potential deadlock between `bdr.autopartition_wait_for_partitions()` and taskmgr|
+| BDR       | 5.3.0   | Bug&nbsp;fix | Block new EPAS automatic partitioning on PGD cluster. |
```

Comment:
A line is missing after this line 25 -

Allow existing automatically partitioned tables when cluster is created or upgraded.

- please check 50-Block-EPAS-autopartitioning-with-BDR file in REL_4_STABLE

Reply:
Fixed